spec/requests/transformation_mappings_spec.rb

```diff
@@ -305,6 +305,79 @@ def href_slug(obj)
           expect(updated_mapping_destination).to match_array(reference_destination)
           expect(transformation_mapping.transformation_mapping_items.count).to eq(TransformationMappingItem.all.count)
         end
+
+        context "can validate vms with csv data and service_template_id are specified" do
+          it "vm belongs to the service_template record" do
+            api_basic_authorize(action_identifier(:transformation_mappings, :validate_vms, :resource_actions, :post))
+            source_ems = FactoryGirl.create(:ems_cluster)
```

Comment:
Is this supposed to be an EMS or a cluster?  Please adjust the factory or variable name accordingly.